ibm-watson-workitem/src/main/java/org/jbpm/process/workitem/ibm/watson/ClassifyImageWorkitemHandler.java

```
@@ -0,0 +1,132 @@
+                        .parameters("{\"owners\": [\"me\"]}")
+                        .build();
+                ClassifiedImage result = service.classify(classifyOptions).execute().getImages().get(0);
+                if (result.getError() != null) {
+                    ErrorInfo errorInfo = result.getError();
+                    logger.error("Error classifying image: " + errorInfo.getDescription());
+                    workItemManager.completeWorkItem(workItem.getId(),
+                                                     null);
+                } else {
+                    List<ClassifierResult> classificationResults = result.getClassifiers();
+                    for (ClassifierResult classification : classificationResults) {
+                        Map<String, String> resultToAdd = new HashMap<>();
+
+                        for (ClassResult classResult : classification.getClasses()) {
+                            resultToAdd.put("classificationname",
+                                            classification.getName());
+                            resultToAdd.put("classname",
+                                            classResult.getClassName());
+                            resultToAdd.put("score",
+                                            String.valueOf(classResult.getScore()));
+                            resultToAdd.put("typehierarchy",
+                                            classResult.getTypeHierarchy());
+                        }
+
+                        widResults.put(RESULT_VALUE,
+                                       resultToAdd);
+                    }
+
+                    workItemManager.completeWorkItem(workItem.getId(),
+                                                     widResults);
+                }
+            } catch (Exception e) {
+                handleException(e);
+            }
+        } else {
+            workItemManager.completeWorkItem(workItem.getId(),
```

I'd fail here as well with "missing image for classification" type of error